### PR TITLE
Move to using AF Indirect Containers.

### PR DESCRIPTION
**Pull request by tpendragon**
This also separates the logic of returning ordered results from just returning
collected items. #ordered_generic_files, for instance, will return all generic
files but in the order asserted via RDF.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,4 +3,4 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in activefedora-aggregation.gemspec
 gemspec
 gem 'byebug' unless ENV['CI']
-
+gem 'active-fedora', :github => "projecthydra/active_fedora"
```
Comment (jcoyne) [inline Gemfile]: Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.

File: lib/active_fedora/aggregation/association.rb [hunk 2]
```diff
@@ -1,60 +1,37 @@
...
 
-    def target_ids=(vals)
-      container.target_ids=(vals)
-    end
+    private
 
-    def target_ids
-      container.target_ids
+    def default_options
+      {:through => default_proxy_class, :foreign_key => :target, :has_member_relation => reflection.predicate, :inserted_content_relation => content_relation}
```
Comment (jcoyne) [inline lib/active_fedora/aggregation/association.rb]: Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.

**Comment by jcoyne** [comment]
Aside from a couple style issues this is great! :+1:

File: lib/active_fedora/aggregation/association.rb (hunk 2 above)
Comment (terrellt) [inline lib/active_fedora/aggregation/association.rb]: Bah. OSU's style guide says 1.8 syntax so it's reflex - I'll switch.

**Comment by terrellt** [comment]
@jcoyne Any complaints with separating #generic_files and #ordered_generic_files ? I thought it'd be good for performance too.

**Comment by jcoyne** [comment]
@terrellt I'm willing to give it a go an see if it works.

**Pull request merged**